src/Http/Routing/perf/Microbenchmarks/Matching/JumpTableMultipleEntryBenchmark.cs

```diff
@@ -27,7 +27,7 @@ public void Setup()
 
         for (var i = 0; i < _strings.Length; i++)
         {
-            _segments[i] = new PathSegment(0, _strings[i].Length);
+            _segments[i] = new PathSegment(1, _strings[i].Length - 2);
```

Comment:
Shouldn't this be `- 1`?

Reply:
I attempted to trim one from the start and one from the end but I don't suppose it matters too much as long as it's a substring.